docs/installation.rst

```diff
@@ -193,13 +193,13 @@ If you already have a dataset on your machine, you can download and deploy the a
 
 .. code-block:: bash
 
-  docker run -e GA4GH_DATA_SOURCE=/data -v /my/ga4gh_data/:/data:ro -d -p 8000:80 --name ga4gh_server ga4gh/server:latest
+  docker run -e GA4GH_DATA_SOURCE=/data -v /my/ga4gh_data/:/data:ro -d -p 8000:80 --name ga4gh_server ga4gh/ga4gh-server:latest
 
 Replace ``/my/ga4gh_data/`` with the path to your data.
 
 This will:
 
-* pull the automatically built image from `Dockerhub <https://registry.hub.docker.com/u/ga4gh/server/>`_
+* pull the automatically built image from `Dockerhub <https://registry.hub.docker.com/u/ga4gh/ga4gh-server/>`_
```

Comment:
Can't say for certain, but it looks like it doesn't.  Do we know who is managing / created the docker image?